### PR TITLE
Added Footer Details

### PR DESCRIPTION
**Pull request by naseerrm**

### CODE REVIEW

File: Models/Blocks/FooterBlock.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+               GroupName = SystemTabNames.Content,
+               Order = 3)]
+        public virtual LinkItemCollection? IntroductionLinks { get; set; }
+
+        [Display(Name = "GetHelp Links",
+             GroupName = SystemTabNames.Content,
+             Order = 4)]
+        public virtual LinkItemCollection? GetHelpLinks { get; set; }
+
+        [Display(Name = "GeneralQueries Lable",
```
Comment (Sowndharya1143) [inline Models/Blocks/FooterBlock.cs:36]: label

File: Models/Blocks/FooterBlock.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+              GroupName = SystemTabNames.Content,
+              Order = 5)]
+        public virtual string? GeneralQueriesLabel { get; set; }
+
+        [Display(Name = "GeneralQueries Details",
+              GroupName = SystemTabNames.Content,
+              Order = 6)]
+        public virtual string? GeneralQueriesDetails { get; set; }
+
+        [Display(Name = "CorporateHQ Lable",
```
Comment (Sowndharya1143) [inline Models/Blocks/FooterBlock.cs:46]: label

File: Models/Blocks/FooterBlock.cs [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+        [Display(Name = "Footer CopyRights",
+               GroupName = SystemTabNames.Content,
+               Order = 10)]
+        public virtual string? FooterCopyRights { get; set; }
+
+        [Display(Name = "Footer Right Links",
+                GroupName = SystemTabNames.Content,
+                Order = 11)]
+        public virtual LinkItemCollection? RightMenu { get; set; }
```
Comment (Sowndharya1143) [inline Models/Blocks/FooterBlock.cs:72]: BasicPagesMenu - use this name

File: Models/Blocks/LinkWithImageBlock.cs [hunk 4]
```diff
@@ -0,0 +1,25 @@
+﻿using EPiServer.Shell.ObjectEditing;
+using EPiServer.Web;
+using System.ComponentModel.DataAnnotations;
+
+namespace employbridge.Models.Blocks
+{
+    [ContentType(
+    DisplayName = "Link image and URL with Description",
+    Description = "Link image and URL with Description",
```
Comment (Sowndharya1143) [inline Models/Blocks/LinkWithImageBlock.cs:9]: AvailableInEditMode = false add this

Comment (naseerrm) [reply Models/Blocks/LinkWithImageBlock.cs:9]: I have added this

File: Models/Blocks/FooterBlock.cs (hunk 3 above)
Comment (naseerrm) [reply Models/Blocks/FooterBlock.cs]: I have updated the name

File: Models/Blocks/FooterBlock.cs (hunk 2 above)
Comment (naseerrm) [reply Models/Blocks/FooterBlock.cs]: I have updated the text

**Review by Sowndharya1143 — APPROVED** [review]

**Pull request merged**